PCU/src/main/java/com/coffeelovers/pcu/model/Product.java

```diff
@@ -0,0 +1,86 @@
+package com.coffeelovers.pcu.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="products")
+public class Product {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private long productID;
+	
+	@Column(name= "materialsID")
+	private long materialsID;
+	
+	@Column(name= "variableCosts")
+	private double variableCosts;
+	
+	@Column(name= "pic")
```

Comment:
please add full name "Picture"
```suggestion
	@Column(name= "Picture")
```